tox.ini

```
@@ -4,27 +4,29 @@ envlist =
     black
     flake8
     isort
-    py35-django111
-    py{36,37,38}-django{111, 22, 30, master}
+    py{36,37,38,39}-dj{22, 30, 31, master}
     docs
 
-[travis]
+[gh-actions]
 python =
     3.5: py35
     3.6: py36
     3.7: py37
-    3.8: py38
+    3.8: py38, black, flake8, isort
+    3.9: py39
 
 [testenv]
 deps =
-    django111: Django~=1.11.0
-    django22: Django>=2.2<3.0
-    django30: Django>=3.0<3.1
-    djangomaster: https://github.com/django/django/archive/master.tar.gz#egg=django
+    dj111: Django~=1.11.0
+    dj22: Django>=2.2,<3.0
+    dj30: Django>=3.0,<3.1
+    dj31: Django>=3.1,<3.2
+    djmaster: https://github.com/django/django/archive/master.tar.gz#egg=django
     coverage
 commands =
     coverage run -m django test --settings=tests.settings {posargs}
     coverage report
+    coverage xml
 
 [testenv:black]
```

There needs to be an ignore rule for the django master branch tests here as well.

```suggestion
[testenv:py{36,37,38,39,py3}-dj{master}]
ignore_errors = true
ignore_outcome = true

[testenv:black]
```